src/Umbraco.Forms.Integrations.Crm.Hubspot/Api/Management/Controllers/ShopifyCompliance/ShopifyComplianceControllerBase.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Cms.Web.Common.Routing;
+using Umbraco.Forms.Integrations.Crm.Hubspot.Filters;
+
+namespace Umbraco.Forms.Integrations.Crm.Hubspot.Api.Management.Controllers.ShopifyCompliance
+{
+    [ApiVersion("1.0")]
```

Comment:
I got a little bit confuse about these ShopifyCompliance endpoints. What should we do with these?

Reply:
I think you've mixed the integration with the  `Umbraco.Cms.Integrations.OAuthProxy` web application, that manages the OAuth flows for all integrations, not `Shopify` in particular.